src/components/Organizations/style.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+export const OrganizationsDiv = styled.div`
+  width: 100%;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 30%);
+  align-items: center;
+  justify-content: center;
+  gap: 3rem 5rem;
+  flex-wrap: wrap;
+
+  margin-top: 3.5rem;
+
```

Comment:
Para alinhar tudo e manter o gap seria interessante colocar um 
```suggestion
margin: 0 auto
```